src/components/Places.js

```diff
@@ -1,38 +1,17 @@
-import React, { Component } from 'react';
-import { GoogleApiWrapper, } from 'google-maps-react';
-import PlacesListItem from './PlacesListItem.js'
+import React from 'react';
```

Comment:
Extracted out Listing into a separate component. Typically you wouldn't have two components in one file